File: core-java-practice/gcr-codebase/KmToMiles.java
import java.util.Scanner;

public class KmToMiles {
    public static void main(String[] args){
        //read input
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the distance in kilometers: ");
        double dist=sc.nextDouble(); //take input as km 
  
        //convert in miles
        double miles=dist * 0.621371;

        System.out.println("The distance in Miles is: " + miles);
        
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/KmToMiles.cs
using System;

class KmToMiles{
    public static void Main(string[] args){

        Console.WriteLine("Enter the distance in km: ");
        double km=double.Parse(Console.ReadLine()); //take input as km 

        //conversion to miles
        double miles=km*0.621371; 
        
        //printing statement
        Console.WriteLine("The distance in Miles is: "+ miles);

    }
}